python/paddle/fluid/__init__.py

```diff
@@ -160,10 +160,9 @@ def __bootstrap__():
             'sync_nccl_allreduce', 'limit_of_tmp_allocation',
             'times_excess_than_required_tmp_allocation'
         ]
-
-    core.init_gflags([sys.argv[0]] +
-                     ["--tryfromenv=" + ",".join(read_env_flags)])
-    core.init_glog(sys.argv[0])
+    program = sys.argv[0] if sys.argv[0].endswith(".py") else ""
```

Comment:
Not all train program endswith `.py` should find a way to identify when calling with `-m`